packages/next-auth/src/authOptions.ts

```diff
@@ -0,0 +1,42 @@
+import { type AuthOptions } from "next-auth";
+import GithubProvider from 'next-auth/providers/github';
+import DiscordProvider from 'next-auth/providers/discord';
+import TwitchProvider from 'next-auth/providers/twitch';
+import { PrismaAdapter } from "@auth/prisma-adapter";
+import { prisma } from "@kampus-db/pano-prisma";
+
+import { env } from "../lib/env";
+
+const prismaAdapter = PrismaAdapter(prisma);
+
+
+export const authOptions: AuthOptions = (
+  {
+    adapter: prismaAdapter,
+    secret: env.SECRET,
+    providers: [
+      {
+        id: 'vercel-email',
+        name: 'No-Reply Kampus',
+        type: 'email',
+        server: '',
+        options: {},
+        async sendVerificationRequest({ identifier: _email, url: _url }) {
+          // Umut look over here for the email :) 
```

Comment:
👋 

I thought we were gonna use the builtin [`EmailProvider`](https://next-auth.js.org/providers/email)

Reply:
We will include our email send here, that's how the customizable works if i understood corrrectly

Reply:
Just to note on this is if we use email provider then we define the server and sender. If we want to use custom email provider this way we can call the API of the custom email provider